Blocks/HTTP/test.cc

```diff
@@ -901,7 +901,7 @@ TEST(HTTPAPI, ServeStaticFilesFrom) {
   EXPECT_EQ("<h1>HTML file</h1>", HTTP(GET(Printf("http://localhost:%d/file.html", FLAGS_net_api_test_port))).body);
   EXPECT_EQ("This is text.", HTTP(GET(Printf("http://localhost:%d/file.txt", FLAGS_net_api_test_port))).body);
   EXPECT_EQ("\211PNG\r\n\032\n", HTTP(GET(Printf("http://localhost:%d/file.png", FLAGS_net_api_test_port))).body);
-  EXPECT_EQ("<h1>NOT FOUND</h1>\n", HTTP(GET(Printf("http://localhost:%d/.DS_Store", FLAGS_net_api_test_port))).body);
```

Comment:
These changes are generally a bad idea. Tests should be dependency-free. In fact, we're testing the resulting string values too.